xml/System.Text.Json/JsonSerializerOptions.xml

```diff
@@ -51,9 +51,18 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Get or sets a value that indicates whether an extra comma at the end of a list of JSON values in an object or array is allowed (and ignored) within the JSON payload being deserialized.</summary>
+        <value><see langword="true" /> if an extra comma at the end of alist of JSON values in an object or array is allowed (and ignored) within the JSON payload being deserialized; <see langword="false" /> otherwise.</value>
```

Comment:
Maybe remove some duplicate text that the summary already has, for brevity.
```suggestion
        <value><see langword="true" /> if an extra comma at the end of a list of JSON values in an object or array is allowed (and ignored); <see langword="false" /> otherwise.</value>
```